File: JAVA-Basic/JavaBasic/src/ForLoop/CleverLily.java
package ForLoop;

import java.util.Scanner;

public class CleverLily {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int age = Integer.parseInt(scan.nextLine());
        double washingMachinePrice = Double.parseDouble(scan.nextLine());
        int toyPrice = Integer.parseInt(scan.nextLine());
        int toysCounter = 0;
        int money = 0;
        double totalMoney = 0;

        for (int i = 1; i <= age; i++) {
            if (i % 2 != 0) {
                toysCounter++;
            } else {
                money += 10;
                totalMoney += money - 1;
            }
        }
        totalMoney += (toyPrice * toysCounter);
        double moneyDiff = Math.abs(totalMoney - washingMachinePrice);

        if (totalMoney >= washingMachinePrice) {
            System.out.printf("Yes! %.2f", moneyDiff);
        } else {
            System.out.printf("No! %.2f", moneyDiff);
        }
    }
}

File: 01.JS-Basic/FOR-ЦИКЪЛ/CleverLily.js
function cleverLily(input) {

    let liliAge = Number(input[0])
    let washingMachinePrice = Number(input[1])
    let toyPrice = Number(input[2])

    let totalMoney = 0
    let toys = 0
    let money = 0
    for (let i = 1; i <= liliAge; i++) {
        if (i % 2 == 0) {
            totalMoney -= 1
            money += 10
            totalMoney += money
        }
        else {
            toys += 1
        }
    }
    if ((totalMoney + toyPrice * toys) >= washingMachinePrice) {
        console.log(`Yes! ${((totalMoney + toyPrice * toys) - washingMachinePrice).toFixed(2)}`)
    }
    else {
        console.log(`No! ${(washingMachinePrice - (totalMoney + toyPrice * toys)).toFixed(2)}`)
    }
}
cleverLily(["21", "1570.98", "3"])